src/icons/pdc.svg

```diff
@@ -0,0 +1,35 @@
+<svg width="27" height="29" viewBox="0 0 27 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Beyond the removal of the height, width and fill, this SVG needs optimised. I use [ImageOptim](https://imageoptim.com)

Reply:
Thanks! Fixed.